bikeshed/__init__.py

```diff
@@ -796,9 +796,21 @@ def decorateAutolink(doc, el, linkType, linkText):
 
 def processIssues(doc):
     import hashlib
-    # Add an auto-genned and stable-against-changes-elsewhere id to all issues.
+    # Add an auto-genned and stable-against-changes-elsewhere id to all issues, and
+    # link to remote issues if possible:
     for el in findAll(".issue:not([id])", doc):
         el.set('id', "issue-"+hashContents(el))
+        remoteIssueID = el.get('data-remote-issue-id')
+        if remoteIssueID:
+            del el.attrib['data-remote-issue-id']
+            githubMatch = re.match(r"([^/]+)/([^#]+)#(\d+)", remoteIssueID)
```

Comment:
This regex overclaims syntax space; it'll match on `Issue(Hahaha, this/looks like a github url#123456789 but it's not.)`.

I think we can get by `r"([\w-]+)/([\w-]+)#(\d+)$"`.